styles/main.css

```diff
@@ -303,11 +306,15 @@ footer hr {
     display: block;
     width: 100%;
     height: 100%;
+    object-fit: cover;
+}
 
+.gallery a:hover {
+    transform: scale(1.1);
 }
 
 .row-span-2 {
-    grid-row: 1 /span 2;
+    grid-row: span 2;
```

Comment:
Using just span 2 allows the element to take its intrinsic position in the grid then span 2 rows/cols from there